.rubocop_todo.yml

```diff
@@ -1,78 +1,213 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-08-24 11:09:20 +0200 using RuboCop version 0.49.1.
+# on 2021-07-06 16:06:45 UTC using RuboCop version 1.12.1.
```

Comment:
Changes in this file are auto-generated and do not need to be reviewed.
https://docs.rubocop.org/rubocop/configuration.html#automatically-generated-configuration